tests/cover/test_sampled_from.py

```diff
@@ -17,11 +17,20 @@
 
 from __future__ import division, print_function, absolute_import
 
+import pytest
+
 from hypothesis import given, settings
-from hypothesis.strategies import sampled_from
+from hypothesis.errors import HypothesisDeprecationWarning
+from hypothesis.strategies import sets, integers, sampled_from
 
 
 @given(sampled_from((1, 2)))
 @settings(min_satisfying_examples=10)
 def test_can_handle_sampling_from_fewer_than_min_satisfying(v):
     pass
+
+
+@given(sets(integers(), min_size=2))
+def test_cant_sample_sets(s):
+    with pytest.raises(HypothesisDeprecationWarning):
+        sampled_from(s).example()
```

Comment:
It's not obvious to me that that's a sensible thing to do TBH. If you're passing a single element set to sampled\_from it's probably a set that can vary in size, and it's weirder to have that sometimes be an error and sometimes not than it is to just make it uniformly an error.